teuthology/task/selinux.py

```diff
@@ -133,6 +133,7 @@ def get_denials(self):
             'comm="ksmtuned"',
             'comm="sssd"',
             'comm="sss_cache"',
+            'scontext=system_u:system_r:NetworkManager_dispatcher_t:s0',
```

Comment:
@djgalloway 
we still see failures like the following:
```
2022-02-24T15:50:11.343 INFO:teuthology.orchestra.run.smithi167.stdout:type=AVC msg=audit(1645716761.044:7707): avc:  denied  { sendto } for  pid=26387 comm="chronyd" path="/run/chrony/chronyc.88830.sock" scontext=system_u:system_r:chronyd_t:s0 tcontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tclass=unix_dgram_socket permissive=1
2022-02-24T15:50:11.343 INFO:teuthology.orchestra.run.smithi167.stdout:type=AVC msg=audit(1645717564.831:11102): avc:  denied  { sendto } for  pid=26387 comm="chronyd" path="/run/chrony/chronyc.154839.sock" scontext=system_u:system_r:chronyd_t:s0 tcontext=system_u:system_r:NetworkManager_dispatcher_t:s0 tclass=unix_dgram_socket permissive=1
```

should we add `scontext=system_u:system_r:chronyd_t:s0` too ?

Reply:
Or just scrap `Xcontext=` entirely.  @b-ranto is this sane?  Do you think we could potentially be missing legit Ceph selinux issues if we whitelist `system_u:system_r:NetworkManager_dispatcher_t:s0`?

Reply:
@guits https://github.com/ceph/teuthology/pull/1717